ansible_collections/arista/avd/roles/eos_designs/templates/network_services/router-ospf.j2

```diff
@@ -0,0 +1,54 @@
+{# Rendering tenant OSPF #}
+{% for tenant in network_services_data.tenants %}
+{%     for vrf in tenant.vrfs %}
+{%         if vrf.ospf is arista.avd.defined and inventory_hostname in vrf.ospf.nodes | arista.avd.default([inventory_hostname]) %}
+{%             set network_services_data.ospf_configured = true %}
```

Comment:
You can set a break after ```ospf_configured``` is set to true, because it has to be true only once. This will save some time if there are 1000's of VRFs. 

Or you can use a different algorithm to save those tenants and respective VRFs in a temporary variable if ```vrf.ospf.enabled is true``` and the other check is also true.